README.md

```diff
@@ -8,10 +8,24 @@ SPDX-License-Identifier: Apache License 2.0
 
 # `libsigopt`
 
-`libsigopt` is a module that contains various optimization utilities.
+`libsigopt` is a module that contains various the core computation elements for [`sigopt-server`](https://github.com/sigopt/sigopt-server) and [`sigoptlite`](https://github.com/sigopt/sigoptlite).
```

Comment:
"contains various computations elements" or 
"contains the core computation elements"